File: app/src/main/java/com/neome/java/api/meta/base/dto/StudioBuildUserSetting.java
// neome.ai API - do not change
//

package com.neome.java.api.meta.base.dto;

import com.neome.java.api.meta.base.Types.EnumDefnUserSettingOptions;
import com.neome.java.api.meta.base.Types.MetaIdVar;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class StudioBuildUserSetting extends StudioBase
{
  @Nullable
  public Boolean doNotShowOnUserSettings;

  @Nullable
  public Boolean doNotShowValueToAdmin;

  @Nullable
  public MetaIdVar optionDataSourceVarId;

  public EnumDefnUserSettingOptions userSettingOptionKind;
}

File: app/src/main/java/com/neome/api/meta/base/dto/StudioBuildUserSetting.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.EnumDefnUserSettingOptions
import com.neome.api.meta.base.Types.MetaIdVar
import com.neome.api.meta.base.dto.StudioBase

interface StudioBuildUserSetting : StudioBase
{
  val doNotShowOnUserSettings: Boolean?
  val doNotShowValueToAdmin: Boolean?
  val optionDataSourceVarId: MetaIdVar?
  val userSettingOptionKind: EnumDefnUserSettingOptions
}